templates/libvirt/xen.xml

```diff
@@ -1,3 +1,8 @@
+{% if vm.audiovm is defined %}
+ {% set audiovm_xid = vm.audiovm.xid %}
+{% else %}
+ {% set audiovm_xid = 0 %}
```

Comment:
Should this really default to 0 (dom0)? I'd say setting audiovm to None should result in some dummy value here (-1? empty?) which then avoids loading the vchan module (so only the null one is loaded). But setting audiovm to something later should still work.

But that's a minor issue. If you prefer to do that later, it's okay, but open an issue to not forget.

Reply:
I'll set it to -1 so I need to adapt the stubdom part to not load module on start.